### DISCUSSION TITLE
D-Brake for Qanba Gravity

### DISCUSSION THREAD

**Discussion by Robear512** [misternoe/cheatbox#15] [Ideas]
I put together a D-brake model that fits Qanba Gravity buttons, mentioned on twitter that I'd like to share it with you. What's the best way to share the STL file to add it to the project?

**Comment by misternoe** [comment]
Sorry for the late response. I think you can attach it here or share via Google Drive or WeTransfer. Im not sure how to do it, but if you have a github repo for this model, there's is a way to contribute to this repository—good option if you see yourself frequently updating the model.

If youd rather I just host it on here, please let me know how you'd like to be credited.

**Comment by Robear512** [comment]
[Dbrake_qanba.zip](https://github.com/misternoe/cheatbox/files/14353716/Dbrake_qanba.zip)
Here's a zip of the full D-brake and the plunger attachment, in case you wanted to make any adjustments.

I'm good with just a name tag or whatever for credit. Honestly I feel like I didn't do much but make the design fit a different plunger, just happy to make it more accessible!

**Reply by misternoe** [reply]
it's all appreciated! Gonna do a test print overnight and I'll try to get this up sometime this week.

**Comment by misternoe** [comment]
Here's how it came out with some Prusa Galaxy Black PETG. It's the equivalent of ~20mm height, so if I manage to find some free time I'll try to build out some shorter variants and possibly a 24mm version. 

![dbrake-quanba](https://github.com/misternoe/cheatbox/assets/3981094/7927aaa4-b9d6-467b-b609-622e64141e43)
![dbrake-quanba-supports](https://github.com/misternoe/cheatbox/assets/3981094/1d1def18-a624-4d34-96b5-16aaa2da13ba)

**Reply by Robear512** [reply]
Awesome! I bet it feels a good bit sturdier in the PETG. I'm just glad to find it doesn't break _immediately_ when printed with PLA.
